src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Telemetry/SearchPageTelemetryEvent.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using NuGet.Common;
+using NuGet.PackageManagement.VisualStudio;
+
+namespace NuGet.PackageManagement.Telemetry
+{
+    /// <summary>
+    /// Represents that a search page was fetched. Similar to <see cref="TelemetryActivity"/> but does not depend on
+    /// <see cref="IDisposable"/> and does not have a start and end time.
+    /// </summary>
+    public class SearchPageTelemetryEvent : TelemetryEvent
+    {
+        public SearchPageTelemetryEvent(
+            Guid parentId,
+            int pageIndex,
+            int resultCount,
+            Stopwatch duration,
```

Comment:
could you just provide the elapsed time here?

Reply:
Yeah, sounds good.